### PR TITLE
Remove FragmentFlowState activeKeys and visibleKeys.

### PR DESCRIPTION
**Pull request by Laimiux**
## What

- Clean up of deprecated functions
- Remove renderer and unnecessary equality check

### CODE REVIEW

File: formula-android/src/main/java/com/instacart/formula/android/internal/FragmentFlowRenderView.kt [hunk 1]
```diff
@@ -119,9 +119,9 @@ internal class FragmentFlowRenderView(
         activity.supportFragmentManager.registerFragmentLifecycleCallbacks(callback, false)
     }
 
-    override val render: Renderer<FragmentFlowState> = Renderer {
```
Comment (Laimiux) [inline formula-android/src/main/java/com/instacart/formula/android/internal/FragmentFlowRenderView.kt:122]: Removing `Renderer` here as equality check on `FragmentFlowState` is not necessary

**Comment by carrotkite** [comment]
### JaCoCo Code Coverage 79.34% :white_check_mark:
| Class | Covered | Meta | Status |
|:---|:---:|:---:|:---:|
| `com/instacart/formula/android/FragmentFlowState` | 0% | 0% | :white_check_mark: |
| `com/instacart/formula/android/internal/FragmentFlowRenderView` | 76% | 0% | :white_check_mark: |


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: formula-android/src/main/java/com/instacart/formula/android/internal/FragmentFlowRenderView.kt (hunk 1 above)
Comment (Jawnnypoo) [reply formula-android/src/main/java/com/instacart/formula/android/internal/FragmentFlowRenderView.kt:122]: Will this have an effect downstream? Should be transparent, right?

**Review by Jawnnypoo — APPROVED** [review]

File: formula-android/src/main/java/com/instacart/formula/android/internal/FragmentFlowRenderView.kt (hunk 1 above)
Comment (Laimiux) [reply formula-android/src/main/java/com/instacart/formula/android/internal/FragmentFlowRenderView.kt:122]: Individual renderers do an equality check by default, so this extra check does nothing

**Pull request merged**